src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/LanguageVersionExtensions.cs

```diff
@@ -10,7 +10,7 @@ internal static class LanguageVersionExtensions
         //
         // This value should be removed when LanguageVersion.CSharp9 is available, and all callers should just
         // reference that constant instead.
-        public static LanguageVersion CSharp9 => LanguageVersion.Preview;
+        public const LanguageVersion CSharp9 = LanguageVersion.Preview;
```

Comment:
i was actually worried abot this being a constant, and not being a static readonly that would pick up the right value if we changed things here.  was that innapropriate on my part?  i.e. i don't want this value hardcoded downstream.

Reply:
It's internal. There is no downstream.